gnome-extra/krb5-auth-dialog/krb5-auth-dialog-43.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson gnome2-utils
+
+DESCRIPTION="An application to acquire and list Kerberos tickets."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/krb5-auth-dialog/"
+SRC_URI="https://download.gnome.org/sources/${PN}/$(ver_cut 1)/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=gui-libs/libadwaita-1.2.0:1
+	dev-libs/glib:2
+	x11-libs/gtk+:3
+	|| (
+		app-crypt/mit-krb5
+		app-crypt/heimdal
+	)
+"
+DEPEND="${RDEPEND}"
+BDEPEND=""
+
+pkg_postinst() {
+	gnome2_schemas_update
```

Comment:
xdg_icon_cache_update missing here and in postrm, so is xdg-utils in inherit

Reply:
`xdg_desktop_database_update` is missing too, so better inherit `gnome2` instead of `gnome2-utils` and remove `pkg_postinst` and `pkg_postrm`, the eclass with take care of it all.  You don't need `xdg-utils` either, it was provided by `gnome2-utils` anyway.

Reply:
Please note in this case order becomes significant - both gnome2 and meson define src_ functions, we are counting on meson being inherited last.

Reply:
done, thanks